### DISCUSSION TITLE
Exporting SQL should have unicode prefix as "N" if DB is Oracle

### DISCUSSION THREAD

**Discussion by Cybersepp** [dbeaver/dbeaver#21864] [Q&A]
This is is similar to the issue below.
https://github.com/dbeaver/dbeaver/issues/11058

I'm using Oracle. Sometimes I have to use "import & export table data using SQL or Database".Source data has EN-DASH "–" character which results as a question mark in the target database. The problem is that there's no 'N' prefix before insertion it seems like. 

For example this is the source table EN_DASH_TEST
![image](https://github.com/dbeaver/dbeaver/assets/73464092/17c7ddc8-1e1a-4b97-8d97-e2c7c8f67053)

This is the preview of how the data will look like in EN_DASH_TEST_BACKUP 
![image](https://github.com/dbeaver/dbeaver/assets/73464092/ea32b00d-5e38-4e38-979e-bf0cdf137991)

This is the data in EN_DASH_TEST_BACKUP after export 
![image](https://github.com/dbeaver/dbeaver/assets/73464092/b32d45c0-bdb9-41b2-a71e-898c4329fcdc)

-     Operating system (distribution) and version: Windows 10
-     DBeaver version 23.2.0.202309041200
-     Database name and version: Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
-     Do you use tunnels or proxies (SSH, SOCKS, etc)? No

Is there a way to add the 'N' prefix before the insert statements?

Thank you in advance!

**Comment by LonwoLonwo** [comment]
Hello @Cybersepp 

Any links on documentation will be helpful.

**Reply by Cybersepp** [reply]
Hey!

I was following this document: https://dbeaver.com/docs/dbeaver/Data-migration/#target-container-and-target-table

**Reply by LonwoLonwo** [reply]
I'm talking about Oracle side docs :)

**Reply by Cybersepp** [reply]
Sorry for late response, 

'NVARCHAR2 Data Type' section talking about the data type itself: https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Data-Types.html#GUID-CC15FC97-BE94-4FA4-994A-6DDF7F1A9904

Oracle docs on using this N prefix: https://docs.oracle.com/en/database/oracle/oracle-database/19/nlspg/programming-with-unicode.html#GUID-9F223401-9892-4380-92FA-656ADCF84B50

**Reply by Cybersepp** [reply]
Hello,

Any update on this? Is there somehow I could help?
